src/workerd/util/sqlite-kv.c++

```diff
@@ -11,6 +11,7 @@ SqliteKv::SqliteKv(SqliteDatabase& db, bool): db(db) {}
 SqliteDatabase& SqliteKv::ensureInitialized(SqliteDatabase& db) {
   // TODO(sqlite): Do this automatically at a lower layer?
   db.run("PRAGMA journal_mode=WAL;");
+  db.run("PRAGMA temp_store=MEMORY;");
```

Comment:
I don't think this is actually needed. There's no problem with SQLite opening temp files on disk in `workerd`. (Unlike the edge runtime, where we already set this pragma anyway.)